lib/watch/index.js

```diff
@@ -48,7 +48,7 @@ const watch = function (args, ignoreWatch, verboseWatch) {
     })
 
     _child.on('exit', function (code, signal) {
-      if (!code) {
+      if (!code === 0) {
```

Comment:
```suggestion
      if (code === 1) {
```

Would this work the same?

Reply:
Seems like this would be more in line with what is intended with the original code, and it also makes sense (not knowing the broader context here).